frontend/app/module/input/page/InputHomePage.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, test } from "vitest";
+
+import { overrideOnce, render, screen } from "app/test/unit";
+
+import { ElectionProvider } from "@kiesraad/api";
+import { electionMock } from "@kiesraad/api-mocks";
+
+import { InputHomePage } from "./InputHomePage";
+
+describe("InputHomePage", () => {
+  overrideOnce("get", "/api/elections/1", 200, { election: electionMock });
```

Comment:
We have an `electionMockData` and a `electionsMockData` in `frontend/lib/api-mocks/ElectionMockData.ts`. They both have an election with id `1`, but their `name`s are different (no differences in other fields). Would be better if the `name`s match.

Reply:
I'll update this in my upcoming PR. I've already been making changes to the mocks

Reply:
I've already pushed a commit. Should I revert it to make the rebase easier?

Reply:
No it's okay, I'll figure it out :+1:

Reply:
@Lionqueen94 is what @jschuurk-kr suggests something the mock changes will cover?

Reply:
Yes that will be done, I'll be showing the election name in the header as per design in the input pages

Reply:
Cool, I'll resolve this

Reply:
> Or just share the same mock object?

Something like this: https://gist.github.com/praseodym/cf70d91d6ea7ecc05138963fbcea363e

Reply:
These 2 commits fixed the name inconsistency. My PR will make the file a lot cleaner including Mark's suggestion